### DISCUSSION TITLE
Need some help on an issue

### DISCUSSION THREAD

**Discussion by Ren-B-7** [altdesktop/playerctl#344] [Q&A]
Playerctl seems to now work in subshells, it detects the player, but doesnt want to interact with it. I dont want to open an issue yet, first want to see if theres a local/ known solution. None of the actions fail, but it just cant play, pause, or interact with players, it can however detect them with ``` playerctl -l ```  

[Screencast from 2024-11-26 15-17-10.webm](https://github.com/user-attachments/assets/7f1376d3-7a0d-4c35-9130-11974dcbe260)

**Comment by ferdnyc** [comment]
I can't reproduce, using `vim` with the `:sh` command to launch a subshell. `playerctl play-pause` operates normally, as does every other command I've tried.

You can use `busctl --json=short --user monitor` to watch D-Bus messages being exchanged, see if maybe there's some difference in the messages sent while using a subshell vs. what's exchanged normally. But if you can see the list of players, then it seems like you have access to the session bus, so I don't know why command messages would be any different from query messages.
